### ISSUE TITLE
ignore unused_field for __mouse__ in generated files

### ISSUE THREAD

**Issue by chalin** [google/pageloader#144]
So that we can avoid this analyzer hint:

```nocode
  hint • The value of the field '__mouse__' isn't used at test/app_po.g.dart:128:19 • unused_field
```

cc @kevmoo @kwalrath

**Comment by chalin** [comment]
@mk13 - do you think that this can be fixed for 3.0.0 final?

At a minimum, an ignore directive should do it:
```dart
  PageLoaderMouse __mouse__; // ignore: unused_field
```

**Comment by mk13** [comment]
Done, will be published with 3.0.0
